docs/source/modules/api.rst

```diff
@@ -19,12 +37,25 @@ API Reference
 
    linear_model.LinearRegression
    linear_model.LogisticRegression
+   linear_model.PoissonRegression
+
+Incremental Learning
+====================
+
+.. currentmodule:: dask_ml
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
    linear_model.PartialPassiveAggressiveClassifier
    linear_model.PartialPassiveAggressiveRegressor
    linear_model.PartialPerceptron
    linear_model.PartialSGDClassifier
    linear_model.PartialSGDRegressor
-   linear_model.PoissonRegression
+   naive_bayes.PartialBernoulliNB
+   naive_bayes.PartialMultinomialNB
```

Comment:
There's also `PartialMiniBatchKMeans` a few lines down if you want to move that up here.